### ISSUE TITLE
[a11y] 9.2.4.2 Meaningful document titles

### ISSUE THREAD

**Issue by tschiebel** [owncloud/web#10732]
audit: https://infinite.owncloud.com/s/PnaAukHGGsLnrMM

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-2-4-2-sinnvolle-dokumenttitel

### Personal, Shares, Spaces, Deleted Files and Account
Note - Both the existing title of the page and the "h1" heading should
be changed to ensure better accessibility and consistency with the navigation and visual
the navigation and visual appearance of the page.

**Comment by pascalwengerter** [comment]
How actionable is this ticket? I'm unsure about what the title&heading should be changed to

**Comment by pascalwengerter** [comment]
Ping @tschiebel @tbsbdr

**Comment by pascalwengerter** [comment]
Discussed with @tbsbdr that this is already fulfilled, closing for now
